grails-app/controllers/org/pih/warehouse/UrlMappings.groovy

```diff
@@ -528,7 +528,7 @@ class UrlMappings {
 
         "/api/replenishments/$id/"(parseRequest: true) {
             controller = { "replenishmentApi" }
-            action = [GET: "read", POST: "create", PUT: "update"]
+            action = [GET: "read", PUT: "update"]
```

Comment:
Change POST to link to "update" action. It's not REST-compliant but it's what a developer would expect if they posted to an endpoint for a resource with an ID.